values/cert-manager/cert-manager-raw.gotmpl

```diff
@@ -150,7 +150,7 @@ resources:
   - apiVersion: v1
     kind: Secret
     metadata:
-      name: otomi-byo-wildcard-cert
+      name: {{ $cm.externallyManagedTlsSecretName }}
```

Comment:
See my other comment. I would just leave this as is. There is no need to change this name in the byo-wildcard-cert right? It would only make it more confusing to use it I think.